js/script.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+$(function() {
+
+    function checkUser(e, elements, users) {
+        setTimeout(function () {
+            elements.cursorPosition = elements.textArea.prop("selectionStart");
+            if(elements.cursorPosition){
+                let value = elements.textArea.val();
+                elements.startPos = getStartPosition(value, elements.cursorPosition-1);
+                if(elements.startPos){
+                    let text = value.substring(elements.startPos, elements.cursorPosition);
+                      return  find(text, users, elements);
+                }
+                return elements.showUsers.empty();
+                }
+        }, 0);
+
+    }
+
+    function getStartPosition(str, pos) {
+        if (!str.substring(pos, pos + 1).match(/[A-Za-z]/)) {
```

Comment:
@lianatumikyan maybe will be easier if you will check this one str.substring(pos, pos + 1) === '@'  -> 
true -> pos + 1
false -> go to else if case

Reply:
@Mariam13  I think that username cannot contain special chars so it means that if we have something like (@jshsjh/d) this it will return position and it's not correct

Reply:
@lianatumikyan  I understood , but on final result you return the position only when str.substring(pos, pos + 1) === '@', so what's the problem if user will be typing (!@#) (it's same as when user will be typing f.e. 'T' (no name which will include this symbol) but nothing will be shown). (FYI username can include numbers too)